autoload/lsp/internal/diagnostics/signs.vim

```diff
@@ -127,15 +127,13 @@ function! s:set_signs(params) abort
 endfunction
 
 function! s:place_signs(server, diagnostics_response, bufnr) abort
+    let l:linecount = get(getbufinfo(a:bufnr)[0],'linecount',0) " linecount available from vim 8.2.0019
```

Comment:
not required for this PR but you can add this check in [healtcheck](https://github.com/prabirshrestha/vim-lsp/blob/master/autoload/health/lsp.vim)